queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+ select * from animals where name like '%mon';
+ select name from animals where date_of_birth between '01-01-2016' and '01-01-2019';
+ select name from animals where neutered = true and escape_attempts <3;
```

Comment:
- [ ] Please make sure to write a query that can list the Date of birth of all animals named either "Agumon" or "Pikachu" because as requested in the project requirements you should list the date of birth of all animals named either "Agumon" or "Pikachu".